Add video module (#72)
vircadia/vircadia-docs-sphinx: README.md

## Using RST

Most of our docs use RST. reStructuredText (RST) is the default plaintext markup language used by Sphinx. It is an extensible markup language, that is fully customizable. To learn more, refer to Sphinx's [reStructuredText Primer](https://www.sphinx-doc.org/en/2.0/usage/restructuredtext/basics.html).
RST should be used for any real documentation, as Markdown only supports very basic directives.
A valuable resource for RST is the [official documentation](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html).

## Using videos

> When adding videos to the documentation, it is important to use h264 *and* vp9 to ensure they can be played it all major internet browsers.

Reviewer: ```suggestion
When adding videos to the documentation, it is important to use h264 *and* vp9 to ensure that they can be played it all major web browsers.
```

To convert videos and animated images to vp9 you can use following command in ffmpeg:

>     ```

Reviewer: ```suggestion

    ```
```
Blank line needed in order for command text to be displayed in its own container.

    ffmpeg -i INPUTFILE -c:v libvpx-vp9 -b:v 0 -crf 5 -vf "scale=-2:'min(600,ih)'" -cpu-used 5 -row-mt 1 -c:a libopus -b:a 96K _static/videos/OUTPUTFILE.webm
    ```
Or to convert to h264:

>     ```

Reviewer: ```suggestion

    ```
```
